src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,54 @@
-export const Tabs = () => {};
+import React from 'react';
+import cn from 'classnames';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void,
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  const handleClickTab = (tab: Tab) => {
+    if (tab.id === selectedTabId) {
+      return;
+    }
+
+    onTabSelected(tab);
```

Comment:
you can save few rows :)
```suggestion
    if (tab.id !== selectedTabId) {
      onTabSelected(tab);
    }   
```